frontends/main/src/app-pages/OnboardingPage/OnboardingPage.tsx

```diff
@@ -169,13 +172,23 @@ const OnboardingPage: React.FC = () => {
       if (activeStep < NUM_STEPS - 1) {
         setActiveStep((prevActiveStep) => prevActiveStep + 1)
       } else {
-        router.push(DASHBOARD_HOME)
+        if (nextUrl) {
+          router.push(nextUrl)
+        } else {
+          router.push(DASHBOARD_HOME)
+        }
       }
     },
     validateOnChange: false,
     validateOnBlur: false,
   })
 
+  useEffect(() => {
+    if (nextUrl) {
+      router.prefetch(nextUrl)
+    }
+  }, [nextUrl])
```

Comment:
Good idea.

I'm with github... I'd add `router` to the prefetch array. It's probably a singleton anyway?